python/cudf/cudf/core/multiindex.py

```diff
@@ -675,12 +687,14 @@ def from_pandas(cls, multiindex):
                 levels=multiindex.levels,
                 codes=multiindex.codes,
                 names=multiindex.names,
+                pandas=multiindex,
             )
         else:
             mi = cls(
                 levels=multiindex.levels,
                 codes=multiindex.labels,
                 names=multiindex.names,
+                pandas=multiindex,
```

Comment:
Instead of adding an additional kwarg to here can we just call `multiindex.to_frame()` in here and then pass that frame to the `source_data` kwarg of the `cls` constructor?